content/en/docs/components/pipelines/sdk/v2/v2-compatibility.md

```diff
@@ -0,0 +1,131 @@
+
+Kubeflow Pipelines SDK v2 compatibility mode lets you use the new pipeline semantics and gain the benefits of logging your metadata to ML Metadata. You can use ML Metadata to help answer questions about the lineage of your pipeline’s artifacts.
+
+To learn more about the work towards Kubeflow Pipelines v2, read the design documents for [Kubeflow Pipelines v2](http://bit.ly/kfp-v2) and [Kubeflow Pipelines v2 compatible
+mode](http://bit.ly/kfp-v2-compatible), or join the Kubeflow Pipelines community.
+
+## Before you begin
+
+1.  Run the following command to install Kubeflow Pipelines SDK v1.6 or higher. If you run this command in a Jupyter notebook, restart the kernel after installing the SDK.
+    
+    ```bash
+    pip install kfp --upgrade
+    ```
+
+1.  Import the kfp and kfp.components packages.
+
+    ```python
+    import kfp
+    import kfp.components as comp
+    ```
+
+1.  Create an instance of the kfp.Client class. To find your Kubeflow Pipelines cluster’s hostname and URL scheme, open the Kubeflow Pipelines user interface in your browser. The URL of the Kubeflow Pipelines user interface is something like https://my-cluster.my-organization.com/pipelines. In this case, the host name and URL scheme are https://my-cluster.my-organization.com.
+
+    ```python
+    # If you run this command on a Jupyter notebook running on Kubeflow, you can
+    # exclude the host parameter.
+    # client = kfp.Client()
+    client = kfp.Client(host='<your-kubeflow-pipelines-host-name>')
+    ```
+
+## Building pipelines using the Kubeflow Pipelines SDK v2
+
+If you are new to building pipelines, read the following guides to learn more about
+using Kubeflow Pipelines SDK v2 to build pipelines and components.
+
+*  [Get started building pipelines using Pipelines SDK v2][build-pipeline].
+*  [Learn how to build pipeline components using Pipelines SDK v2][build-component].
+*  [Build lightweight Python function-based components using Pipelines SDK
+   v2][python-component].
+
+If you are familiar with building Kubeflow pipelines, the Kubeflow Pipelines SDK v2 
+introduces the following changes:
+
+*   The following changes affect how you build components:
+
+    *   All component inputs and outputs must be annotated with their data type.
+
+    *   The Kubeflow Pipelines SDK v2 makes a distinction between inputs and outputs that
+        are _parameters_ and those that are _artifacts_.
+
+        *   Parameters are inputs or outputs of type `str`, `int`, `float`, or `double`
+            that typically are used to change the behavior of a pipeline. Input parameters
+            are always passed by value, which means that they are inserted into the
+            command used to execute the component.
```

Comment:
We can also add that parameters are always stored in MLMD, which means they can be queried later.

Reply:
Done.